app/config/providers.php

```diff
@@ -85,7 +85,11 @@
         'enabled' => true,
         'mock' => false,
     ],
-
+    'discord' => [
+        'developers' => 'https://discordapp.com/developers/docs/topics/oauth2',
+        'icon' => 'icon-discord',
+        'enabled' => true,
```

Comment:
Just need to add here the mock>false attribute. It's new just added it today.

Reply:
ok, I'll do that